pkgs/development/libraries/mapbox-gl-native/default.nix

```diff
@@ -1,20 +1,38 @@
-{ lib, mkDerivation, fetchFromGitHub, cmake, pkg-config
-, qtbase, curl, libuv, glfw3 }:
+{ lib, mkDerivation, fetchFromGitHub, fetchpatch, cmake, pkg-config
+, qtbase, curl, libuv, glfw3, rapidjson }:
 
 mkDerivation rec {
   pname = "mapbox-gl-native";
-  version = "2020.06.07";
+  version = "1.6.0";
 
   src = fetchFromGitHub {
     owner = "mapbox";
     repo = "mapbox-gl-native";
-    rev = "e18467d755f470b26f61f6893eddd76ecf0816e6";
-    sha256 = "1x271gg9h81jpi70pv63i6lsa1zg6bzja9mbz7bsa4s02fpqy7wh";
+    rev = "maps-v${version}";
+    sha256 = "027rw23yvd5a6nl9b5hsanddc44nyb17l2whdcq9fxb9n6vcda4c";
     fetchSubmodules = true;
   };
 
+  patches = [
+    (fetchpatch {
+      name = "add-support-for-using-qmapboxgl-as-proper-cmake-dependency.patch";
```

Comment:
This commit is actually called "Add support for qmapboxgl installation"

I think we should also add ["add-support-for-using-qmapboxgl-as-proper-cmake-dependency](https://github.com/mapbox/mapbox-gl-native/pull/16591/commits/ab27b9b8207754ef731b588d187c470ffb084455) from the same PR as the first patch too: with it, I believe we'll be able to update to the latest mapbox-gl-qml version, switch to building mapbox-gl-qml with cmake and drop some hacks.

Reply:
Oops, I took the wrong commit. I'll fix it later.